view/frontend/requirejs-config.js

```diff
@@ -2,29 +2,29 @@ var config = {
     map   : {
         '*': {
             // Magento FE libs
-            'algoliaCommon'       : 'Algolia_AlgoliaSearch/internals/common',
-            'algoliaAutocomplete' : 'Algolia_AlgoliaSearch/autocomplete',
-            'algoliaInstantSearch': 'Algolia_AlgoliaSearch/instantsearch',
-            'algoliaInsights'     : 'Algolia_AlgoliaSearch/insights',
-            'algoliaHooks'        : 'Algolia_AlgoliaSearch/hooks',
+            'algoliaCommon'       : 'Algolia_AlgoliaSearch/js/internals/common',
+            'algoliaAutocomplete' : 'Algolia_AlgoliaSearch/js/autocomplete',
+            'algoliaInstantSearch': 'Algolia_AlgoliaSearch/js/instantsearch',
+            'algoliaInsights'     : 'Algolia_AlgoliaSearch/js/insights',
+            'algoliaHooks'        : 'Algolia_AlgoliaSearch/js/hooks',
 
             // Autocomplete templates
```

Comment:
@damcou If we have to move `template` anyway (which will disrupt downstream customization of Autocomplete and Recommend) then it makes sense to me to move this outside of `internals` as well. These file will be regularly customized and referenced so it never really made sense to me to include them under `internals`. 

We also already have a `template` directory (currently for an unused backend render implementation I think). Having two `template` directories might be a little confusing for some. What do you think?

Reply:
This makes total sense your right, adding the change 👌